game/engine.py

```
@@ -297,6 +297,20 @@ def _resolve_engine_path(cls):
         """Return the first available engine entrypoint for this platform."""
         for path in cls.ENGINE_CANDIDATES:
             if os.path.exists(path):
+                # Serverless workaround: copy the Linux C++ binary to the writable /tmp directory
+                # and restore the execute (+x) permission.
+                if os.name != 'nt' and not path.endswith('.py'):
+                    tmp_path = '/tmp/checkora_main'
```

Ruff (0.15.14)

[error] Probable insecure usage of temporary file or directory: "/tmp/checkora_main"

(S108)